### PR TITLE
nagios-check_logfiles version bump

### PR DESCRIPTION
**Pull request by nvinson**

### CODE REVIEW

**Comment by gktrk** [comment]
@gentoo/proxy-maint

**Comment by nvinson** [comment]
Andrey Grozin independently updated sbcl.  Removing from pull request.

File: net-analyzer/nagios-check_logfiles/nagios-check_logfiles-3.7.5.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+KEYWORDS="~alpha ~amd64 ~ppc ~ppc64 ~sparc ~x86"
+
+RDEPEND=">=net-analyzer/nagios-plugins-1.4.13-r1"
+
+S="${WORKDIR}/${MY_P}"
+
+src_configure() {
+	econf \
+		--libexecdir=/usr/$(get_libdir)/nagios/plugins \
```
Comment (SoapGentoo) [inline net-analyzer/nagios-check_logfiles/nagios-check_logfiles-3.7.5.2.ebuild]: `--libexecdir="${EPREFIX}"/usr/$(get_libdir)/nagios/plugins \`

File: net-analyzer/nagios-check_logfiles/nagios-check_logfiles-3.7.5.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+		--sysconfdir=/etc/nagios
```
Comment (SoapGentoo) [inline net-analyzer/nagios-check_logfiles/nagios-check_logfiles-3.7.5.2.ebuild]: same here

**Pull request closed without merging**